flake8_adjustable_complexity/checker.py

```diff
@@ -68,3 +68,17 @@ def run(self) -> Generator[Tuple[int, int, str, type], None, None]:
                 self._error_message_template.format(actual_complexity, max_expected_complexity),
                 type(self),
             )
+
+    @classmethod
+    def add_options(cls, parser) -> None:
+        parser.add_option(
+            '--max-mccabe-complexity',
```

Comment:
Also, info about the argument should be added to readme.

Reply:
Done)